src/common/darktable.c

```diff
@@ -413,6 +414,14 @@ int dt_init(int argc, char *argv[], const int init_gui, lua_State *L)
   }
 #endif
 
+  if(!dt_openmp_init_stacksize())
+  {
+    fprintf(stderr, "[dt_init] your system uses a low default stacksize and we could not set\n");
+    fprintf(stderr, "[dt_init] the OMP_STACKSIZE environment variable to %dB.\n", SAFESTACKSIZE);
+    fprintf(stderr, "[dt_init] Please set the OMP_STACKSIZE environment variable before starting darktable\n");
+    return 1;
+  }
+
```

Comment:
Ok, maybe create a config variable? SAFESTACKSIZE is just defined to what ulimit -Ss is on my system (Fedorara 23). I've also seen 2M or 4M while googling! With musl libc's 80K darktable doesn't even start on my system.